ext/drx/drx.h

```diff
@@ -435,6 +435,19 @@ DR_EXPORT
 size_t
 drx_buf_get_buffer_size(void *drcontext, drx_buf_t *buf);
 
+
+DR_EXPORT
+/**
+ * Pads a basic block with a label for routines which rely on inserting instrumentation
+ * after every instruction. Note that users of this routine must act on the previous
+ * instruction before the is-app check afterwards, because the label is not marked as an
+ * app instruction.
+ *
+ * \returns whether terminator was introduced.
```

Comment:
suggest: s/terminator/label/, and also explain up above that the label won't be added in certain situations (btw, would it be simpler and more general to just always add it, in case some client wants to take an action after an instr and somehow it's legal after a cti or syscall?  Does DR complain about a label being added after a syscall?)